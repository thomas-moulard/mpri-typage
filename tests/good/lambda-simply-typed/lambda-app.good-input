val app_int_fun (f : int -> int) (x : int) = 
  f x
