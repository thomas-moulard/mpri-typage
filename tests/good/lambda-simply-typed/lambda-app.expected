app_int_fun = fun (f : int -> int).fun (x : int).f x

app_int_fun : ((int -> int) -> (int -> int))
